css/css-quiz.md

```diff
@@ -2414,3 +2414,9 @@ input.invalid {
 - [x] How to display HTML elements
 - [ ] How to save HTML elements
 - [ ] How to make HTML elements
+
+    #### Q160. Which of the following statements is TRUE for CSS?
```

Comment:
```suggestion
#### Q160. Which of the following statements is TRUE for CSS?
```

Reply:
@VipulDhillon please look at the comment, if possible make updates in the next PR, thanks